src/test/groovy/ExampleSpec.groovy

```diff
@@ -20,10 +27,78 @@ class ExampleSpec extends BaseSpec {
             navigateToPracticeFormPage()
             PracticeFormPage practiceFormPage = at PracticeFormPage
         when: "Fill out the form"
-
-
             sleep(30)
```

Comment:
This was not merged with the latest master. Please always use the latest master in your branches.